docs/_pages/faq.md

```diff
@@ -96,23 +96,42 @@ server {
 ```
 
 ## How to keep map data after reboot
-In default the robot will lose map data after each reboot. To prevent the map reset use the mirobo command line tool (same tool you use for the installation process) and execute `mirobo raw-command set_lab_status 1`.
-This can be also done via filesystem access `echo -n "1" > /mnt/data/rockrobo/lab.cfg` on the robot itself.
+
+By default, the robot will lose map data after each reboot. For Gen2 devices,
```

Comment:
I'm not sure how this should be accomplished for gen1 devices :thinking: 

Is backing up and restoring `/mnt/data/rockrobo/last_map` enough? I haven't tested this to be sure.

Reply:
Afaik it's not possible at all

Reply:
That's unfortunate :/ I'll resolve this then.